packages/orbit-components/src/Loading/__tests__/index.test.js

```diff
@@ -1,37 +1,38 @@
 // @flow
 import * as React from "react";
-import { shallow } from "enzyme";
+import { render, screen } from "@testing-library/react";
 
 import Loading from "../index";
 import TYPE_OPTIONS from "../consts";
 
 describe("Loading", () => {
-  const dataTest = "test";
-  it("should render type", () => {
-    const type = TYPE_OPTIONS.BOX_LOADER;
-    const component = shallow(<Loading type={type} />);
-    const loader = component.find("Loading__StyledLoader");
-    expect(component.find("Loading__StyledLoader").childAt(0).parent()).toEqual(loader);
-  });
-  it("should render type with text", () => {
+  const { BOX_LOADER, SEARCH_LOADER, INLINE_LOADER } = TYPE_OPTIONS;
+  it("should have expected DOM output", () => {
     const text = "Test test";
-    const type = TYPE_OPTIONS.BOX_LOADER;
-    const component = shallow(<Loading type={type} text={text} />);
-    const loader = component.find("Loading__StyledLoader");
-    const loadingText = component.find("Loading__StyledLoadingText").render();
-    expect(component.find("Loading__StyledLoader").childAt(0).parent()).toEqual(loader);
-    expect(loadingText.text()).toBe(text);
+    const dataTest = "test";
+    render(<Loading type={BOX_LOADER} text={text} dataTest={dataTest} />);
+    expect(screen.getByTestId(dataTest)).toBeInTheDocument();
+    expect(screen.getByText(text)).toBeInTheDocument();
   });
-
-  it("should have data-test", () => {
-    const component = shallow(<Loading dataTest={dataTest} />);
-    expect(component.render().prop("data-test")).toBe(dataTest);
+  it("should have children", () => {
+    render(<Loading>kek</Loading>);
+    expect(screen.getByText("kek")).toBeInTheDocument();
   });
 
-  it("should render text", () => {
-    const text = "Test test";
-    const component = shallow(<Loading text={text} />);
-    const loadingText = component.find("Loading__StyledLoadingText").render();
-    expect(loadingText.text()).toBe(text);
+  it.each(Object.values({ BOX_LOADER, SEARCH_LOADER, INLINE_LOADER }))(
+    "should have circled icons %s",
+    type => {
+      render(<Loading type={type} dataTest="kek" />);
+      expect(screen.getByTestId("kek").firstChild).toHaveStyle({
+        display: "flex",
+        justifyContent: "center",
+      });
+    },
+  );
+
+  it("should have spinner icon", () => {
+    render(<Loading dataTest="kek" type={TYPE_OPTIONS.PAGE_LOADER} />);
+    expect(screen.getByTestId("kek").firstChild).toHaveStyle({ width: "40px", height: "40px" });
+    expect(screen.getByTestId("kek").querySelector("svg")).toHaveAttribute("viewBox", "0 0 40 40");
```

Comment:
Tests should be as agnostic as possible and not have hard coded values if they are irrelevant. Here I think it's enough to just check whether an `svg` element exists, because it's the only one. If we happen to change the spinner illustration with a different `viewBox`, then the test won't break.
```suggestion
    expect(screen.getByTestId("kek").querySelector("svg")).toBeInTheDocument();
```